sidecar/config/config.go

```diff
@@ -66,8 +66,9 @@ type Config struct {
 	Registry   Registry   `yaml:"registry"`
 	Controller Controller `yaml:"controller"`
 
-	Supervise bool     `yaml:"supervise"`
-	App       []string `yaml:"app"`
+	Supervise    bool     `yaml:"supervise"`
+	App          []string `yaml:"app"`
+	HealthChecks []string `yaml:"healthchecks"`
```

Comment:
Need to consider if we want each healthcheck to be a simple string, or an healthcheck object.
For example:
```yaml
healthchecks:
	- type:	 	http
	  value: 	http://localhost:9080/health
	  interval: 10
	  
	- type:   	http
	  value:  	http://localhost:9080/health2
	  method: 	POST
	  code:   	201
  	  interval: 30
	  
	- type: 	command
	  value: 	/bin/sh -s "cat /opt/myapp/app.log | tail -n 1000 | grep -i ERROR"
	  intreval: 60
	  
	- type:		tcp
	  value: 	localhost:5555
	  timeout: 	3
	  interval: 30  
```

* Not saying we should implement all these customizations, but maybe leave room for?

For flags / environment variables, we can support only a subset of the customizations, so that it won't get too cumbersome to define. We can expect / drive users to define the healthchecks via the config file, baked into their app images.